extension/js/common/ui/key-import-ui.ts

```diff
@@ -39,6 +40,7 @@ export class KeyCanBeFixed extends Error {
 export class UserAlert extends Error {}
 
 export class KeyImportUi {
+  public readonly emailDomainsToSkip = ['yahoo', 'live', 'outlook'];
```

Comment:
@sosnovsky Fixed all comments.
However just curious, why do we skip those domains for rendering alias?

Reply:
I found old issue for skipping these domains - https://github.com/FlowCrypt/flowcrypt-browser/issues/1885

Reply:
Got it. Thank you for your reference